views/artist-search-result.hbs

```diff
@@ -0,0 +1,16 @@
+<h1>Here are your results!</h1>
+
+{{#each info}}
+  <ul>
```

Comment:
you can show image of each album here, you need to check if image exist, then show them:
``{{#if this.images.0.url}}
            <img
              src={{this.images.0.url}}
              alt={{this.name}}
            />
            {{/if}}